src/views/OrderPage/OrderPage.tsx

```diff
@@ -0,0 +1,51 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import * as Style from './index';
+import { Header } from '../../shared/components';
+import * as Components from '../../modules/order';
+import Topping from 'modules/order/models/Topping';
+
+const OrderPage: React.FC = () => {
+  const [price, setPrice] = useState(0.0);
+
+  const toppings = [
+    { name: 'Chilli', emoji: '🌶', price: 1 } as Topping,
+    { name: 'Corn', emoji: '🌽', price: 1 } as Topping,
+    { name: 'Egg', emoji: '🥚', price: 1 } as Topping,
+    { name: 'Pineapple', emoji: '🍍', price: 2 } as Topping,
+    { name: 'Meat', emoji: '🍢', price: 4 } as Topping,
+    { name: 'Shrooms', emoji: '🍄', price: 2 } as Topping,
+    { name: 'Bacon', emoji: '🥓', price: 2 } as Topping,
```

Comment:
Do not use `as Topping`, that is casting. Instead, define the variable as `const toppings: Topping[] = [...]`